### PR TITLE
Inomurko/end2end gen swagger

### PR DESCRIPTION
**Pull request by InoMurko**
A continuation of https://github.com/omisego/elixir-omg/pull/1161#pullrequestreview-323160614
https://github.com/omisego/elixir-omg/pull/1161#issuecomment-558270051

## Overview

Deleted openapi generated source code so that it's created on demand from yaml files.

## Changes

- Change 1

## Testing

Describe how to test your new feature/bug fix and if possible, a step by step guide on how to demo this.

### CODE REVIEW

File: .formatter.exs [hunk 1]
```diff
@@ -5,11 +5,11 @@
     "mix.exs",
     "apps/*/mix.exs",
     "apps/*/{lib,test,config}/**/*.{ex,exs}",
```
Comment (pnowosie) [inline .formatter.exs]: Looks similar to these added below (:question: )

File: priv/.gitignore [hunk 2]
```diff
@@ -29,3 +29,11 @@ erl_crash.dump
 elixir-*.tar
 
 .elixir_ls/
+
+#generated applications and source
```
Comment (pnowosie) [inline priv/.gitignore]: Oh then - having them here - why we need to ignore these in the top-most `.gitignore` again?
Also I'd simplify it to just: `apps/`

File: .gitignore [hunk 3]
```diff
@@ -54,4 +54,9 @@ docker-compose.override.yml
 .sobelow
 
 #ganache end2ned
-data/
\ No newline at end of file
+data/
+
+#generated applications and source
+priv/apps/child_chain_api
```
[the remark this answers is not in the document]
Comment (InoMurko) [reply .gitignore]: no, because you don't want to ignore all subapps, just generated ones!

[the remark this answers is not in the document]
Comment (InoMurko) [reply .gitignore]: not really, you want to ignore just the generated applications! itest is still there and you would need to commit these changes

**Pull request merged**